lib/apartment.rb

```diff
@@ -124,6 +125,12 @@ def extract_tenant_config
     rescue ActiveRecord::StatementInvalid
       {}
     end
+
+    def active_record_log
+      return @active_record_log if defined?(@active_record_log)
```

Comment:
you can use `lib/apartment/railtie.rb` to set the default value.
I think we should default to false because if we default to true, when the users update the gem, they will start having more output than they used to, which might not be their desired behavior. Generally I'm in favor of keeping the current behavior, as the default.

Reply:
no problem!

Reply:
I did change the default behavior to false